inc/telemetry.class.php

```diff
@@ -144,6 +144,15 @@ static public function grabWebserverInfos() {
       // check if host is present (do no throw php warning in contrary of get_headers)
       if (filter_var(gethostbyname(parse_url($CFG_GLPI['url_base'], PHP_URL_HOST)),
           FILTER_VALIDATE_IP)) {
+
+          // Issue #3180 - disable SSL certificate validation (wildcard, self-signed)
+          stream_context_set_default([
```

Comment:
`allow_self_signed` seems missing (see equivalent code in GlpiMailer class)

Reply:
I tried this code on https://self-signed.badssl.com/ without issue.
Have you more details ?
Regards

Reply:
Only the php doc: http://docs.php.net/manual/en/context.ssl.php#context.ssl.allow-self-signed :)

Reply:
@trasher Thanks.
> allow_self_signed boolean 
> Allow self-signed certificates. Requires verify_peer. 

For me, as verify_peer is set to false, this option is bypassed (ie : no verification of the SSL certificate is made).

Reply:
Maybe yes, this is a bit unclear

Reply:
checked with a self-signed cert on my localhost instance, it's ok with proposed fix (and so without allow_self_signed tag)